keyboards/senselessclay/had60/info.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This is not a valid info.json, and appears to be a via json file.  The via json files don't belong in this repo, and need to be PR'ed to the `the-via/keyboards` repo, instead. 

https://docs.qmk.fm/#/reference_info_json